docs/sources/upgrading/upgrade-considerations.md

```diff
@@ -0,0 +1,1000 @@
+---
```

Comment:
The contents of this file are copied from `docs/sources/upgrading/_index.md`?
If so, we should please rather do a `git mv` so that the history is preserved.